resources/migrations/000_migrations.yaml

```diff
@@ -14300,6 +14300,22 @@ databaseChangeLog:
             constraintName: fk_action_model_id
             onDelete: CASCADE
 
+  - changeSet:
+      id: v47.00-004
+      author: qnkhuat
+      comment: Added 0.47.0 - Add auto_incremented to metabase_fielj
+      changes:
+        - addColumn:
+            tableName: metabase_field
+            columns:
+              - column:
+                  name: database_auto_incremented
+                  type: boolean
+                  remarks: Indicates this field is auto incremented
+                  defaultValueBoolean: false
```

Comment:
consulted with QP team on whether or not we should have a default value: https://metaboat.slack.com/archives/C04DN5VRQM6/p1678112076860729.

we decided we should have `false` as default